### ISSUE TITLE
2.3.5 does not use @JsonSerialize inherited from meta annotation or package annotation

### ISSUE THREAD

**Issue by trask** [immutables/immutables#465]
Changing

```
public boolean isJacksonProperties() {
  if (declaringType().isPresent()) {
    return declaringType().get().jacksonSerializeMode() != JacksonMode.NONE;
  }
  return isJacksonSerialized();
}
```

to

```
public boolean isJacksonProperties() {
  if (declaringType().isPresent()
      && declaringType().get().jacksonSerializeMode() != JacksonMode.NONE) {
    return true;
  }
  return isJacksonSerialized();
}
```

fixes for me, though not sure it is correct.  Thanks.

https://github.com/immutables/immutables/blob/immutables-2.3.5/value-processor/src/org/immutables/value/processor/meta/Proto.java#L1613

**Comment by elucash** [comment]
yep, just fixed in ec8a4d6d4e64e36648ef1b3ef36087e6c750d8d8
thank you!

**Comment by elucash** [comment]
Released in 2.3.7
